odoo/addons/base/models/ir_attachment.py

```diff
@@ -341,6 +341,9 @@ def _postprocess_contents(self, values):
                     else:  # datas
                         img = ImageProcess(base64.b64decode(values['datas']), verify_resolution=False)
 
+                    if not img.image:
+                        raise UserError(_("Empty source, SVG, or WEBP"))
```

Comment:
@ryce-odoo You also need to import `_` for this to work.
Maybe you'll want to add a test to check this scenario too ?

Reply:
@bso-odoo _ is already imported in this file. Not sure if it is worth it to add a test case for a change as small as this as it would involve adding an additional image file to the test folder in base. If you insist I can add it, but it seems excessive.

Reply:
@ryce-odoo It is indeed imported at the file level, but I get the following error when I try to upload a WEBP imager enamed to JPG:
```
  File "/home/odoo/src/enterprise/documents/models/ir_attachment.py", line 76, in create
    attachments = super().create(vals_list)
  File "<decorator-gen-59>", line 2, in create
  File "/home/odoo/src/odoo/odoo/api.py", line 415, in _model_create_multi
    return create(self, arg)
  File "/home/odoo/src/odoo/odoo/addons/base/models/ir_attachment.py", line 643, in create
    values = self._check_contents(values)
  File "/home/odoo/src/odoo/odoo/addons/base/models/ir_attachment.py", line 377, in _check_contents
    values = self._postprocess_contents(values)
  File "/home/odoo/src/odoo/odoo/addons/base/models/ir_attachment.py", line 345, in _postprocess_contents
    raise UserError(_("Empty source, SVG, or WEBP"))
TypeError: 'str' object is not callable
```
because `_` is hidden by a string in this scope:
```py
_type, _, _subtype = mimetype.partition('/')
```
Since this small change manages to fail, I believe a test could be useful. :wink:

I guess a test could be simply to create an attachment using the data of any of the existing SVG files in web (or just put one in a string within the test), but forcing the file name and the mimetype to wrong values.